scripts/deploy_contracts.js

```diff
@@ -71,9 +72,13 @@ async function deployContracts(deployData) {
   if(needsDeploy(deployData.ethernaut)) {
 		console.log(deployData);
     console.log(`Deploying Ethernaut.sol...`);
-    ethernaut = await Ethernaut.new(props)
-    console.log(colors.yellow(`  Ethernaut: ${ethernaut.address}`));
-    deployData.ethernaut = ethernaut.address;
+    try {
+      ethernaut = await Ethernaut.new(props)
+      console.log(colors.yellow(`  Ethernaut: ${ethernaut.address}`));
+      deployData.ethernaut = ethernaut.address;
+    } catch (err) {
+      console.log('Error deploying Ethernaut: ' + err)
```

Comment:
Is it the intention to continue running or should the process be aborted if this fails?

Reply:
I added this for debugging, I'll remove it